front/src/app/account/new-order/new-order.component.ts

```diff
@@ -450,6 +464,22 @@ export class NewOrderComponent implements OnInit, OnDestroy {
     return `${orderItem.id}_${Order.getProductLabel(orderItem)}`;
   }
 
+  updateAllDataFormats() {
+    this.isOrderPatchLoading = true;
+    const dataFormatName = this.orderItemFormGroup.get('formatsForAll')?.value || '';
+    for (const item of this.currentOrder.items) {
+      const availableFormats = item.available_formats || [];
+      if (availableFormats.indexOf(dataFormatName) > -1) {
```

Comment:
@kalbermattenm 
Here, when customer chooses a format, if the format is not available for the current item, nothing happens. The customer will have to specify it manually if he wants to submit order.